ipywidgets/widgets/widget.py

```diff
@@ -302,8 +302,9 @@ def seperate_buffers(substate, path, buffer_paths, buffers):
         # any part of the state that has buffers needs to be treated seperately
         # since of a issue as indicated in .open(..)
         for key in set([k[0] for k in buffer_paths]):
-            state_with_buffers[key] = state[key]
-            del state[key]
+            if key in state: # if we didn't remove it allready
+                state_with_buffers[key] = state[key]
+                del state[key]
```

Comment:
actually, i see it's a set now.. so I have no ideas why the del statement every failed